packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -60,6 +67,26 @@ export const ArticlePostCard = forwardRef(function PostCard(
     );
   }
 
+  const renderOverlay = () => {
+    if (isFeedPreview) {
+      return null;
+    }
+
+    if (!shouldUseFeedLayoutV1) {
```

Comment:
I personally prefer using positive conditions as it seems more readable, in this case I would go better with 
`if (shouldUseFeedLayoutV1) {
return (
      <Link href={post.commentsPermalink}>
        <CardLink
          title={post.title}
          onClick={onPostCardClick}
          href={post.commentsPermalink}
        />
      </Link>
    );
return <CardButton title={post.title} onClick={onPostCardClick} />;
}`

Nothing blocking, just a matter of preference.

Reply:
I do defensive programming which returns early, one of the main points is to reduce indents to make it easier to read and follow. The more complicated (multi-line) the return is, the more indent will be applied. In this case, I leave the most complicated return on the bottom-most part to not have the additional indent of the if statement.

Kind of a personal preference, I guess. But if there is a strong point for better devex, I'd be happy to apply changes.

Reply:
I'm also a fan of defensive programming and returning early, I vote for keeping it as is...